src/com/hms_networks/americas/sc/time/TimeConstants.java

```diff
@@ -0,0 +1,39 @@
+package com.hms_networks.americas.sc.time;
+
+/**
+ * Class of constants for the HMS MU Americas Solution Center time library for Ewon Flexy devices.
+ *
+ * @since 1.0
+ * @author HMS Networks, MU Americas Solution Center
+ */
+public class TimeConstants {
+
+  /**
+   * Default username of Ewon Flexy devices. Used as the default username in time utilities unless
+   * otherwise specified by the user.
+   */
+  public static final String DEFAULT_DEVICE_USERNAME = "adm";
+
+  /**
+   * Default password of Ewon Flexy devices. Used as the default password in time utilities unless
+   * otherwise specified by the user.
+   */
+  public static final String DEFAULT_DEVICE_PASSWORD = "adm";
+
+  /** Date format generated by the local time offset HTML file in result files. */
+  public static final String TIME_OFFSET_DATE_FORMAT = "dd/MM/yyyy HH:mm:ss";
+
+  /** Folder where time offset files should be stored on the filesystem. */
+  public static final String TIME_OFFSET_FILE_LOCATION = "/usr/TimeOffset/";
```

Comment:
Should this be called path rather than location?

Reply:
I think 'folder' might be better. What do you think? 

It would be TIME_OFFSET_FILE_FOLDER

Reply:
Since it contains /usr it is more correct to call it a path as a path contains the directory hierarchy where as a folder would just be /TimeOffset

Reply:
ok

Reply:
I think  TIME_OFFSET_DIRECTORY_PATH would be the most descriptive.

Reply:
That way its clear that its a path to the time offset folder/dir

Reply:
changed to TIME_OFFSET_DIRECTORY_PATH